core/pom.xml

```diff
@@ -88,6 +88,7 @@
                         <arg>${compilerArg0}</arg>
                         <arg>${compilerArg1}</arg>
                         <arg>${compilerArg2}</arg>
+                        <arg>-XDignore.symbol.file</arg>
```

Comment:
why?

Reply:
to remove javac warnings:

```
[INFO] Compiling 1692 source files to /Users/marregui/workspace/questdb_other/core/target/classes
[WARNING] /Users/marregui/workspace/questdb_other/core/src/main/java/io/questdb/ServerMain.java:[51,15] Signal is internal proprietary API and may be removed in a future release
[WARNING] /Users/marregui/workspace/questdb_other/core/src/main/java/io/questdb/std/ConcurrentHashMap.java:[2138,37] Unsafe is internal proprietary API and may be removed in a future release
[WARNING] /Users/marregui/workspace/questdb_other/core/src/main/java/io/questdb/griffin/engine/functions/columns/ColumnUtils.java:[33,22] Unsafe is internal proprietary API and may be removed in a future release
[WARNING] /Users/marregui/workspace/questdb_other/core/src/main/java/io/questdb/std/Unsafe.java:[43,33] Unsafe is internal proprietary API and may be removed in a future release
[WARNING] /Users/marregui/workspace/questdb_other/core/src/main/java/io/questdb/std/Unsafe.java:[56,38] Unsafe is internal proprietary API and may be removed in a future release
[WARNING] /Users/marregui/workspace/questdb_other/core/src/main/java/io/questdb/std/Unsafe.java:[58,30] Unsafe is internal proprietary API and may be removed in a future release
[WARNING] /Users/marregui/workspace/questdb_other/core/src/main/java/io/questdb/std/Unsafe.java:[214,26] Unsafe is internal proprietary API and may be removed in a future release
[WARNING] /Users/marregui/workspace/questdb_other/core/src/main/java/io/questdb/std/Unsafe.java:[308,43] Unsafe is internal proprietary API and may be removed in a future release
[WARNING] /Users/marregui/workspace/questdb_other/core/src/main/java/io/questdb/ServerMain.java:[295,34] Signal is internal proprietary API and may be removed in a future release
[WARNING] /Users/marregui/workspace/questdb_other/core/src/main/java/io/questdb/ServerMain.java:[295,16] Signal is internal proprietary API and may be removed in a future release
[WARNING] Signal is internal proprietary API and may be removed in a future release
```

we know about this, it just looks ugly when we build the project.